src/main/java/com/bookmanagement/BookManagement/entity/Author.java

```diff
@@ -0,0 +1,24 @@
+package com.bookmanagement.BookManagement.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Author {
```

Comment:
Generally, we add the suffix `entity` while naming them.

Reply:
Done